src/python/bcc/__init__.py

```diff
@@ -495,7 +495,7 @@ def load_funcs(self, prog_type=KPROBE):
 
         return fns
 
-    def load_func(self, func_name, prog_type, device = None):
+    def load_func(self, func_name, prog_type, device = None, attach_type = 38):
```

Comment:
Can we define a call BPFAttachType to define some <name, value> pairs? Using 38 is kind of cryptic.

Reply:
Thanks a lot for your feedback.

Could you explain what changes you have in mind? I did not understand what you are suggesting.

BTW, 38 is `MAX_BPF_ATTACH_TYPE`.

Reply:
MAX_BPF_ATTACH_TYPE => -1?